akka-projection-slick/src/main/scala/akka/projection/slick/internal/SlickOffsetStore.scala

```diff
@@ -94,10 +108,23 @@ import slick.jdbc.JdbcProfile
       offsetStr: String,
       manifest: String,
       mergeable: Boolean,
-      lastUpdated: Instant)
+      lastUpdated: Long)
 
   val offsetTable = TableQuery[OffsetStoreTable]
 
-  def createIfNotExists: Future[Done] =
-    db.run(offsetTable.schema.createIfNotExists).map(_ => Done)(ExecutionContexts.parasitic)
+  def createIfNotExists: Future[Done] = {
+    val prepareSchemaDBIO = SimpleDBIO[Unit] { jdbcContext =>
+      val connection = jdbcContext.connection
+      dialect.createTableStatements.foreach(statement => connection.prepareStatement(statement).execute())
```

Comment:
Better if we close the prepared statement. We have a utility for it, check `JdbcSessionUtil.tryWithResource`. 

We can also use a simple `statement` instead of `prepareStatement`

If will look like:

```scala
tryWithResource(connection.statement(statement)) { stmt =>  
 stmt.execute()
}
```